hack/verify-dep.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+# Copyright 2019 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -euo pipefail
+
+echo "Verifying dep check"
+dep check
```

Comment:
if `dep check` failed, should it `exit $errorno`? otherwise travis ci still passes?

Reply:
`dep check` returns non zero code if dependencies are not satisfied, which will make Travis exit. We don't to check for return code explicitly.